core/root/config/templates/stack-template.json

```diff
@@ -2,9 +2,45 @@
   "AWSTemplateFormatVersion": "2010-09-09",
   "Description": "kube-aws Kubernetes cluster {{.ClusterName}}",
   "Resources": {
+    "ASGNotificationTarget": {
+      "Type": "AWS::SQS::Queue"
+    },
```

Comment:
I see that we aren't using this queue at all for listening incoming lifecycle events but polling via `aws autoscaling describe-auto-scaling-instances` to detect state transitions of ec2 instances, right?
Assuming my understanding is correct, I guess we'd better add some comment on this resource, so that we can clearly see that this resource is not actually used anywhere from code but just passed to the asg lifecycle resources in the control-plane stack and node pool stacks as required.
Also, naming this resource like `DummyASGNotificationT` would even better clarify the fact?